src/Support/Helpers.php

```diff
@@ -418,6 +419,17 @@ public function data_get($target, $key, $default = null)
                 $target = $target[$segment];
             } elseif (is_object($target) && isset($target->{$segment})) {
                 $target = $target->{$segment};
+            } elseif (is_object($target) && is_string($segment)) {
+                $mirror = new ReflectionClass(get_class($target));
```

Comment:
dit vind ik wel spannend, is dit van een versie van de echte Helpers class in illuminate/support?

Reply:
Nee. De ->where deed het niet op de collectie, ik heb dat nu zonder ->where geschreven en deze wijziging eruit gehaald.